api/proto/solaris/v1/solaris.proto

```diff
@@ -62,6 +62,7 @@ message AppendRecordsRequest {
 // AppendRecordsResult contains the number or records added to the log
 message AppendRecordsResult {
   int64 added = 1;
+  repeated Record records = 2;
```

Comment:
Two concerns:
1. Please add a flag to the AppendRecordRequest, which will indicate that the Result should contain the records with IDs. Returning records in the AppendResult may double the trafic unnecessary in case of writer doesn't care about the IDs ...
2. What is the reason to get the whole payload of Record back, only if the ID is needed. Please consier return IDs only, not the records in the Append Result

Reply:
1. Added field FirstID to specify id of first inserted record. Caller can use Query(StartID: FirstID) to fetch list of records. This field returned allways since it do not have overhead
2. Added flag returnIDs to explicitly return list of IDs
3. Changed list of records to list of string IDs

Reply:
I did not get what is the firstID is. If you presume to use the same response for different API calls, better to make a new object, or what?

Reply:
I would suggest the following behavior which makes the API cleaner:
1. If client doesn't care about inserted records IDs, no flag in the request is specified
2. If client cares, the flag is provided and the result contains all the new IDs